jetty-websocket/websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/internal/ExtensionStack.java

```diff
@@ -62,9 +62,17 @@ public ExtensionStack(WebSocketComponents components, Behavior behavior)
 
     public void close()
     {
-        for (Extension e : extensions)
+        for (Extension ext : extensions)
         {
-            e.close();
+            try
+            {
+                ext.close();
+            }
+            catch (Throwable t)
+            {
+                if (LOG.isDebugEnabled())
+                    LOG.debug("Extension Error During Close", t);
+            }
```

Comment:
```suggestion
            IO.close(ext);
```

Reply:
This is fine as long as no runtime exceptions are thrown as `IO.close()` will not catch them.